src/packet_distribution.cpp

```diff
@@ -703,6 +703,16 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
     w_courtroom->on_authentication_state_received(authenticated);
   }
 
+ //AssetURL Packet
+  else if (header == "ASS") {
+    if (f_contents.size() > 1) { // This can never be more than one link.
```

Comment:
doesn't this crash if I just send `ASS#%`

Reply:
Yes and fixed.